src/core/Akka/IO/TcpListener.cs

```diff
@@ -71,9 +71,9 @@ private IEnumerable<SocketAsyncEventArgs> Accept(int limit)
             {
                 var self = Self;
                 var saea = new SocketAsyncEventArgs();
-                saea.Completed += (s, e) => self.Tell(e);
+                saea.Completed += (s, e) => self.Tell(new SocketEvent(e));
```

Comment:
Wrap naked SocketAsyncEventArgs in a struct

Reply:
:+1: